testcontainers/src/core/wait/log_strategy.rs

```diff
@@ -30,6 +30,13 @@ impl LogWaitStrategy {
         Self::new(LogSource::StdErr, message)
     }
 
+    /// Create a new [`LogWaitStrategy`] that waits for the given message to appear in either
+    /// standard output logs or standard error logs.
+    /// Shortcut for `LogWaitStrategy::new(LogSource::BothStd, message)`.
+    pub fn both_std(message: impl AsRef<[u8]>) -> Self {
```

Comment:
`both` seems to be applicable only to the source of log, but here it might be confusing (like "and" semantic - wait for a message in both)

`stdout_or_stderr` seems straightforward here (or `any_std` at least)

WDYT?